docs/user/alerting/pre-configured-connectors.asciidoc

```diff
@@ -85,3 +112,19 @@ A message indicates that this is a preconfigured connector.
 
 [role="screenshot"]
 image::images/pre-configured-connectors-view-screen.png[Pre-configured connector view details]
+
+If preconfigured connector belong to the preconfigured action type, it is disabled for the details preview.
```

Comment:
I don't think it matters if the connector is a preconfigured action type.  All preconfigured connectors have their details preview disabled.  So I think this can be shorted to:

> The connector details preview is disabled for preconfigured connectors.